prusa/connect/printer/models.py

```diff
@@ -155,6 +155,36 @@ def __repr__(self):
                 f" [{self.source}], {data}")
 
 
+class Snapshot(LoopObject):
+    """Snapshot from the camera"""
+
+    endpoint = "/c/snapshot"
+    method = "PUT"
+    needs_token = True
+
+    # pylint: disable=too-many-arguments
+    def __init__(self, camera_fingerprint: str, camera_token: str, data: bytes,
```

Comment:
I told you that data is most important, please move it to first argument position!